test/protocol/clients/BosonVoucher.js

```diff
@@ -9,7 +9,6 @@ const Role = require("../../../scripts/domain/Role");
 const { DisputeResolverFee } = require("../../../scripts/domain/DisputeResolverFee");
```

Comment:
I think this file should be renamed `BosonVoucherTest.js` to match the pattern of al the other tests.

Reply:
done